docs/docsite/rst/reference_appendices/interpreter_discovery.rst

```diff
@@ -3,10 +3,13 @@
 Interpreter Discovery
 =====================
 
+Note: the behavior of this option changed in ansible-core 2.17. Consult the
+previous documentation if you are using ansible-core < 2.17.
+
 Most Ansible modules that execute under a POSIX environment require a Python
 interpreter on the target host. Unless configured otherwise, Ansible will
-attempt to discover a suitable Python interpreter on each target host
-the first time a Python module is executed for that host.
+attempt to discover a suitable Python interpreter on each target host the first
+time a Python module is executed for that host.
```

Comment:
Can you revert this change? Did an IDE automatically impose a max line length here or something?

Reply:
I did it manually. It seems the document has somewhat of a 80-chars limit, so I thought I'd just fix it as well. Especially [this line](https://github.com/ansible/ansible-documentation/blob/eade223e4464b9f72902f79d8f9da3bf3b78b57d/docs/docsite/rst/reference_appendices/interpreter_discovery.rst?plain=1#L21) seems way out of line.

I will revert it back.